chatstripe/payment/serializers.py

```diff
@@ -0,0 +1,28 @@
+from datetime import time
+from rest_framework import serializers
+from .models import CardDetail
+from django.contrib.auth.models import User
+from rest_framework.exceptions import ValidationError
+from django.utils import timezone
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['id', 'first_name', 'last_name', 'email', 'username', 'is_active', 'is_staff']
+        # write_only_fields = ['id']
+
+class CardDetailSerializer(serializers.ModelSerializer):
+    user = UserSerializer(read_only=True)
+    class Meta:    
+        model = CardDetail
+        fields = '__all__'
+
+    
+    def validate_expiry_of_card(self, date_value):
```

Comment:
add validators on models, there might be the case when we are adding values in the model without using the serializer,
in that case, validators will validate the values provided in the model.